processor/tailsamplingprocessor/README.md

```diff
@@ -429,4 +429,83 @@ As a rule of thumb, if you want to add probabilistic sampling and...
 
 **A.** This is likely a load issue. If the collector is processing more traces in-memory than the `num_traces` configuration
 option allows, some will have to be dropped before they can be sampled. Increasing the value of `num_traces` can
-help resolve this error, at the expense of increased memory usage.
\ No newline at end of file
+help resolve this error, at the expense of increased memory usage.
+
+## Monitoring and Tuning 
+
+See [metrics.go][metrics_go] for the full list metrics available for this component.
+
+### Dropped Traces
+
+A circular buffer is used to ensure the number of traces in-memory doesn't exceed `num_traces`. When a new trace arrives, the oldest trace is removed. This can cause a trace to be dropped before its sampled. To reduce the chance of this happening, either increase `num_traces` or decrease `decision_wait`. Both of those options increase memory usage.
+
+**Number of Traces Dropped**
+```
+otelcol_processor_tail_sampling_sampling_trace_dropped_too_early
+```
+
+**Pre-emptively Preventing Dropped Traces**
+
+A trace will be dropped without sampling if it's removed from the circular buffer before `decision_wait`.
+
+To track how long traces remain in the buffer use:
+```
+otelcol_processor_tail_sampling_sampling_trace_removal_age
+```
+
+It may be useful to calculate latency percentiles like p1 and compare that value to `decision_wait`. Values close to `decision_wait` are at risk of being dropped if trace volume increases.
+
+**Slow Sampling Evaluation**
+```
+otelcol_processor_tail_sampling_sampling_decision_timer_latency
+```
+This measures latency of sampling a batch of traces and passing sampled traces through the remainder of the collector pipeline. A latency exceeding 1 second can delay sampling decisions beyond `decision_wait`, increasing the chance of traces being dropped before sampling.
```

Comment:
I didn't actually test this but i'm assuming that latency >1 second would cause some 1 second ticks from `time.Ticker` to be ignored. And maybe this is too much of an edge case to include?

Reply:
godoc for time.Ticker:

```
// NewTicker returns a new Ticker containing a channel that will send
// the current time on the channel after each tick. The period of the
// ticks is specified by the duration argument. The ticker will adjust
// the time interval or drop ticks to make up for slow receivers.
// The duration d must be greater than zero; if not, NewTicker will
// panic. Stop the ticker to release associated resources.
```

We don't do any async processing during the tick, so, I assume this is indeed the case.